lib/clientBuilder.js

```diff
@@ -243,11 +243,9 @@ proto.fingerprint = function(options) {
       return deferred.resolve(msg.fingerprint);
     }
     if (msg.type === 'FingerprintServiceReady') {
-      if (iframe.contentWindow) {
-        iframe.contentWindow.postMessage(JSON.stringify({
-          type: 'GetFingerprint'
-        }), sdk.options.url);
-      }
+      e.source.postMessage(JSON.stringify({
```

Comment:
is ```e.source``` always defined? Or should we check for undefined just in case?

Reply:
It appears to be available on every [`Event`](https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent/source).

The only case where it _could_ be `undefined` is if the event itself is `underfined`, which will get caught [here](https://github.com/okta/okta-auth-js/blob/0a5b9d87eb43535ab0fba605cbc3385f642f0551/lib/clientBuilder.js#L231).

Reply:
Sounds good. I also found this info about [the dispatched event](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage#The_dispatched_event), which as you said, means we will have a ```source```